src/services/refactors/moveToFile.ts

```diff
@@ -167,6 +171,18 @@ registerRefactor(refactorNameForMoveToFile, {
         if (!interactiveRefactorArguments) {
             return emptyArray;
         }
+        const endPosition = context.endPosition;
+        if (endPosition) {

```

Comment:
I would explicitly test for `undefined` - in the (weird, rare?) case where an editor sends a 0-length span at position 0, you won't get a refactoring offered here.

Reply:
Right thank you for picking up on it, 0 is also falsy